lib/utils/ws-requestor.js

```diff
@@ -95,7 +95,7 @@ class WsRequestor extends BaseRequestor {
     if (type === 'session:reconnect') payload = this._sessionData;
     assert.ok(url, 'WsRequestor:request url was not provided');
 
-    const msgid = short.generate();
+    const msgid = this.call_sid;
```

Comment:
what is the purpose of this change?  I asked for test cases, not a change to functionality

Reply:
There is usecase:

client --connect (data with msgID1)--> wss
client <-- open ---- wss

client <--close -----wss
client  ----reconnect (data with msgID2)--> wss
client <---- message (ACK with msgID2)--- wss

client timeout due to ACK message contains MSGID2 which is not match with msgID1.

I see the the WsRequestor keep generating new msgId everytime it send data to WSS, but the class still expecting the ACK message contains msg of initial request.

Reply:
The reconnect **should** have a new msgid.  Your fix is to re-use the initial msgid again.  I think the proper fix is to properly match to msgid2 in the reconnect